src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -530,4 +530,19 @@ export class BrowsingContextImpl {
       },
     };
   }
+
+  async addPreloadScript(
+    params: Script.AddPreloadScriptParameters
+  ): Promise<Script.AddPreloadScriptResult> {
+    const result = await this.#cdpTarget.cdpClient.sendCommand(
+      'Page.addScriptToEvaluateOnNewDocument',
+      {
+        source: params.expression,
```

Comment:
Need to use `params.sandbox` somewhere.

Would it map to `worldName` in the CDP param?

Reply:
yes

Reply:
Done